recipes/tf_object_detection/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "tf_object_detection" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/Maxar-Corp/ratpac/releases/download/od-{{ version }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
I’m not comfortable with this source: there is no code documenting how this tarball is built, and it is not the official source. I’d much prefer a recipe that pulls from the official source and documents the build steps to turn that into a conda package as part of the conda recipe dir.

Reply:
@hajapy it would be great the pull the tar.gz from the source it self but the [object detection API](https://github.com/tensorflow/models/tree/master/research/object_detection) is still a folder with a bunch of scripts and not as installable software. I w'd rather have the team at tensorflow package and release the tar file but as you can see they haven't. Thats why I moved the folder to a place where the team I work with can actively support the files.

Reply:
as far as` a code documenting how this tarball is built`, I'm happy to do that. If you have suggestion on how to do this better, please let me know.

Reply:
I think the value of getting this into CF outweighs waiting for when/if google ever turns the objdet api into a pip package, so I support your effort here. I'm hoping we can find a better way to do that such that the code visibly comes from the official source and the mechanism by which it turns into a conda package is codified in the recipe.

1. All the steps you followed to build the tarball in the first place should be converted into a build.sh and added to this recipe. (I imagine this involves: contributing a setup.py so it can be packaged, building the protos, including the slim package somehow). The setup.py can be added using a patch.

2. The source url should point to https://github.com/tensorflow/models, though you may want to point to a recent sha off of master rather than the outdated tags.

3. Versioning will be weird. Since there are no pip packages to match, you could do our own versioning (perhaps starting at 0.0.1). 


A brief search also brought up some other efforts to address this:

This fork adds a setup.py and was released to pypi. Not official, nor up to date. 
https://github.com/brandonschabell/models/tree/master/research/object_detection
https://pypi.org/project/object-detection

An issue requesting this feature: https://github.com/tensorflow/models/issues/4887 and a corresponding PR exemplifying how to convert into pip installable distribution: https://github.com/tensorflow/models/pull/4889

Reply:
@hajapy great. I will continue to explore this: writing up the `built.sh` and tagging `sha` off of master.

Reply:
@asimshankar or @karmel would you help the community bring TF_object_detection to Conda-Forge. It w'd be challenging without release of an official tarball and if you can assist in any way, it w'd be greatly appreciated. There was some discussion on creating a `pip` installable file [last year](https://github.com/tensorflow/models/issues/4887) but is no progress since. I'll be glad to assist in any way possible.

Reply:
@hajapy hope all is good. can you explain what you mean by "contributing a setup.py so it can be packaged"? correct me if I'm wrong but using a patch for the setup.py w'd require an PR accepted by Google so that setup.py.patch file indexed and add the setup file using `git -- diff setup.py`. any thoughts?

Reply:
@scopatz. I hope the thread answered some of the question you raised in reviewing this. 

@hajapy asked the source url of this PR be directly connected to google rather than what I have on my github but it seems that tensorflow developers are not going to release a tarball anytime soon. 
I reached out to some and looked into previous PRs on Tensorflow and they don't seem to be interested in packaging the object detection and slim API. I'm happy to move the source when they release it but in the mean time, can I push a PR where the source is connected to my github. any thoughts?

Reply:
Are there objections to merging this?

Reply:
The user `Andrew Ashkar` raised his concern on the source url. But he mentioned that he understand the challenge here. Google doesn't release the tarballs. 

@conda-forge/tensorflow Is there any objection to creating a PR for Object detection and Slim off of a personal account (at least for now) since Google doesn't (won't) release tarballs for these files? Please see the thread for detail